### PR TITLE
Comment deletion fix

### PR DESCRIPTION
**Pull request by viktor992**
Check if nodeAfter/nodeBefore of current position exists in getNearbyHighlights before continuing.

### CODE REVIEW

File: client/components/comment.vue [hunk 1]
```diff
@@ -58,7 +58,7 @@
                   flat
                   icon
                   small
-                  @click.stop="onDeleteClicked"
+                  @click="onDeleteClicked"
```
Comment (mitar) [inline client/components/comment.vue]: Why this change? I like removal of `stop`, because stopping propagation is generally problematic (because often one depends on propagation somewhere else, like to close modals). But why is this change necessary here?

Comment (mitar) [inline client/components/comment.vue]: Why this change? I like removal of `stop`, because stopping propagation is generally problematic (because often one depends on propagation somewhere else, like to close modals). But why is this change necessary here?

Comment (viktor992) [reply client/components/comment.vue]: I realized that if a comment is not focused and you click on those buttons it keeps defocused. A comment should be focused if you click on it (including the buttons).

Comment (mitar) [reply client/components/comment.vue]: Yea. That is in general exactly the case why never use this. :-)
